pom.xml

```diff
@@ -163,28 +163,38 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
                 <version>${cdi-version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>jakarta.ejb</groupId>
+                        <artifactId>jakarta.ejb-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>jakarta.el</groupId>
+                        <artifactId>jakarta.el-api</artifactId>
+                    </exclusion>
+                </exclusions>
             </dependency>
             <dependency>
-                <groupId>javax.ws.rs</groupId>
-                <artifactId>javax.ws.rs-api</artifactId>
+                <groupId>jakarta.ws.rs</groupId>
+                <artifactId>jakarta.ws.rs-api</artifactId>
                 <version>${jaxrs-version}</version>
```

Comment:
If memory serves me well, we talked in the past about making the dependencies “provided” scope. Does that (still) apply and do we want to take that into this release?

Reply:
Good memory, @ederks85!  This was discussed on our mailing list and Ken opened the following Issue #158.  He also assigned this to the [MicroProfile 4.0 milestone](https://github.com/eclipse/microprofile/milestone/5), so we should include this type of update.  Thanks!

Reply:
I provided the changes for #158

Reply:
My recommendation  is  to release milestone 1 as-is and to address this in the next  milestone or for the RC release. Sound OK?

Reply:
Sounds good to me, it is not blocking any M or RC release

Reply:
I would rather do this work for M1.  This could possibly affect some applications.  It would be good to flesh this out as part of a Milestone vs an RC.

Reply:
At the platform level only or also for component specs?

Reply:
Platform level would be sufficient for this M1.

Reply:
Merged #158.  So, this item is now resolved.